firmware/application/freqman.cpp

```diff
@@ -114,35 +114,38 @@ bool load_freqman_file_ex(std::string& file_stem, freqman_db& db, bool load_freq
     char* line_end;
     std::string description;
     rf::Frequency frequency_a, frequency_b;
-    char file_data[257];
+    const int read_buf_size = 128;
+    char file_data[read_buf_size + 1];
     freqman_entry_type type;
     freqman_index_t modulation = 0;
     freqman_index_t bandwidth = 0;
     freqman_index_t step = 0;
     freqman_index_t tone = 0;
 
-    db.clear();
+    //db.clear();
+    //db.shrink_to_fit():
+    std::vector<freqman_entry>().swap(db);
```

Comment:
Why the swap? clear and shrink_to_fit should do what you want if you're trying to release the memory.

Reply:
And if it doesn't maybe leave a comment here for future devs?

Reply:
I searched a lot around the c++ vector memory usage. Only swap guarantee an empty vector. 
I will add a comment about them